activemq/datadog_checks/activemq/data/conf.yaml.example

```diff
@@ -12,16 +12,6 @@ init_config:
     #
     collect_default_metrics: true
 
-    ## @param new_gc_metrics - boolean - optional - default: False
-    ## Set to true to use better metric names for garbage collection metrics.
-    ## jvm.gc.cms.count   => jvm.gc.minor_collection_count
-    ##                       jvm.gc.major_collection_count
-    ## jvm.gc.parnew.time => jvm.gc.minor_collection_time
-    ##                       jvm.gc.major_collection_time
-    ## The default value is false to ensure backward compatibility.
-    #
-    # new_gc_metrics: false
-
```

Comment:
This is on master:
https://github.com/DataDog/integrations-core/blob/87e6f39df8981fc73ea3fe741c60aed467aa2581/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/init_config/jmx.yaml#L13-L24

Reply:
Thanks! I think this is fixed. Let me know.

Reply:
I think this belongs in `init_config/jmx.yaml` https://github.com/DataDog/integrations-core/blob/87e6f39df8981fc73ea3fe741c60aed467aa2581/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/init_config/jmx.yaml#L13-L24